packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -271,14 +271,21 @@ The following parameters can be posted as a part of the URL-encoded form values
 | Parameter               | Description                                                                                                                                                                                                                                                                                                                        | Type   |
 | :---------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----- |
 | code                    | Required if `grant_type` is `authorization_code`. The value is what was returned from the [authorization endpoint](#authorize). The code has a lifetime of 60 seconds.                                                                                                                                                             | String |
-| code_verifier           | Required if `grant_type` is `authorization_code`  and `code_challenge` was specified in the original `/authorize` request. This value is the code verifier for [PKCE](#parameter-details). Okta uses it to recompute the `code_challenge` and verify if it matches the original `code_challenge` in the authorization request.     | String |
-| grant_type              | Can be one of the following: `authorization_code`, `password`, `client_credentials`, or `refresh_token`. Determines the mechanism Okta uses to authorize the creation of the tokens.                                                                                                                                               | String |
+| code_verifier           | Required if `grant_type` is `interaction_code`, or if the `grant_type` is `authorization_code`  and `code_challenge` was specified in the original `/authorize` request. This value is the code verifier for [PKCE](#parameter-details). Okta uses it to recompute the `code_challenge` and verify if it matches the original `code_challenge` in the authorization request.     | String |
```

Comment:
for interaction code we have original request not to `/authorize` but is either `/token` or `/interact`

Reply:
Required if `grant_type` is `interaction_code`, or if the `grant_type` is `authorization_code` and `code_challenge` was specified in the original `/authorize` request. This value is the code verifier for [PKCE](#parameter-details). Okta uses it to recompute the `code_challenge` and verify if it matches the original `code_challenge` in the authorization, interact, or initial token request.

This description is getting a little long, so I'm tempted to move some of the details to the Parameter details section. @susanharper-okta thoughts?

Reply:
I think it's ok. Not any less meaty than the code_verifier description. And I think meaty is good. :)